Readme.md

```diff
@@ -10,6 +10,14 @@ Currently missing features:
  * discovery messages (receiving and sending)
  * correct stream termination with stream_termination bit (although most devices are
   not supporting this on the receiver site)
+## Setup
+To install the package use pip3. Go to the folder where your setup.py is located and execute
+
+    ``pip install .``
```

Comment:
Only one backtick at the beginning and the end!
https://guides.github.com/features/mastering-markdown/